SDWebImage/SDImageLoadersManager.m

```diff
@@ -88,7 +88,7 @@ - (BOOL)canLoadWithURL:(nullable NSURL *)url {
     NSArray<id<SDImageLoader>> *loaders = self.loaders;
     UNLOCK(self.loadersLock);
     for (id<SDImageLoader> loader in loaders.reverseObjectEnumerator) {
-        if ([loader respondsToSelector:@selector(loadImageWithURL:options:context:progress:completed:)]) {
```

Comment:
@dreampiggy , I don't know wether we should reserve this check, `SDImageLoader` has only `@required` methods, but `Xcode` can also compile successfully even implement them, if we reserve, I think we should also do this check on `canLoadWithURL`.

Reply:
@zhongwuzw Seems just by practice. Because Objective-C's `@required` protocol will not trigger a compile error for most of project configuration (until you specify `-Werror-objc_protocol`)...

I think this check can be replaced with that `canLoadWithURL`, which may be better.  This one can be merged now.